### DISCUSSION TITLE
[Q&A] 现在插件商店需要科学网络才能显示吗？

### DISCUSSION THREAD

**Discussion by ravenk1n9** [lobehub/lobehub#888] [General Question] [answered]
![Screenshot_20240101-091228.png](https://github.com/lobehub/lobe-chat/assets/85299267/87a053a4-b3ea-44a8-a71f-933ebfa437b5)

插件商店获取失败，开启科学上网后能正常访问

**Comment by arvinxx** [comment]
https://mping.chinaz.com/speedtest/chat-plugins.lobehub.com 可以看下测速

**Reply by ravenk1n9** [reply, marked as answer]
![Screenshot_20240103-231533.png](https://github.com/lobehub/lobe-chat/assets/85299267/1495629a-4fe6-4310-9468-a7909b663b44)

![Screenshot_20240103-000400.png](https://github.com/lobehub/lobe-chat/assets/85299267/6243a906-d0ad-49f8-a317-f77fa44c4a6a)

这两天测试都有多个超时，我宽带和4g都是江苏电信

**Reply by arvinxx** [reply]
那估计可能有被污染了的情况
